packages/vscode-extension/src/codeLensProvider.ts

```diff
@@ -146,23 +146,33 @@ export class PlaceholderCodeLens extends vscode.CodeLens {
 export class CryptoCodeLensProvider implements vscode.CodeLensProvider {
   private userDataRegex: RegExp;
   private localDebugRegex: RegExp;
+  private envSecretRegex: RegExp;
 
   constructor() {
     this.userDataRegex =
       /fx-resource-[a-zA-Z\-]+\.[a-zA-Z\-_]+(?:Secret|Password|VariableParams)=(.*)/g;
     this.localDebugRegex =
       /(?: *|\t*)"(?:clientSecret|SimpleAuthEnvironmentVariableParams|botPassword)": "(crypto_.*)"/g;
+    this.envSecretRegex = /#?[a-zA-Z\-_]+(?:SECRET|PASSWORD)=(crypto_.*)/g;
```

Comment:
Our convention for secret names is SECRET_ prefiex.

Reply:
Thanks, updated.